src/test/java/uk/gov/hmcts/ccd/v2/internal/controller/UICaseControllerTest.java

```diff
@@ -135,4 +145,39 @@ void shouldPropagateExceptionWhenThrown() {
                 () -> caseController.getCaseEvent(CASE_REFERENCE, EVENT_ID.toString()));
         }
     }
+
+    @Nested
+    @DisplayName("GET /internal/cases/{caseId}/events")
```

Comment:
Hmm I thought it would be external like in description of the ticket:

v1 endpoint: https://github.com/hmcts/ccd-data-store-api/blob/master/src/main/java/uk/gov/hmcts/ccd/endpoint/std/EventsEndpoint.java#L34-L53

v2: GET /cases/{cid}/events

so should likve in v2/external package and URL should not have internal prefix?

Reply:
The controller should not have the UI prefix then too.

Reply:
Changed to external